lib/discover/file.js

```diff
@@ -0,0 +1,20 @@
+var fs = require('fs');
+
+module.exports.discover = function fileDiscover(urlObj, original, callback) {
+    var filename = urlObj.host + urlObj.pathname;
+    fs.readFile(filename, function (err, data) {
+        if (err) {
+            return callback(err);
```

Comment:
It's typically better to use the style:

```
callback(err);
return;
```

Though it's one more line, it's less error prone.